js/stdlib/ts/ALGO.ts

```diff
@@ -2426,6 +2426,30 @@ const balanceOf = async (acc: Account | Address, token?: Token): Promise<BigNumb
   return (await balanceOfM(acc, token || null)) || bigNumberify(0);
 };
 
+const doAccountAppOptedIn = async (nacc: NetworkAccount, ctcId: ContractInfo): Promise<boolean> => {
+  const ls = await getLocalState_(nacc.addr, ctcId);
+  return ls !== undefined;
+}
+const accountAppOptedIn = async (acc: Account, ctc: ContractInfo): Promise<boolean> => {
```

Comment:
This should take `Account | Address`, because `Account` has a signer and the whole point of this is to be able to inspect other accounts that you don't control, not just to have a different calling pattern